Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/DataContentViewerOtherCases.java

```diff
@@ -80,7 +85,9 @@
     "DataContentViewerOtherCases.toolTip=Displays instances of the selected file/artifact from other occurrences.",})
 public class DataContentViewerOtherCases extends JPanel implements DataContentViewer {
 
-    private final static Logger logger = Logger.getLogger(DataContentViewerOtherCases.class.getName());
+    private static final long serialVersionUID = -1L;
+
+    private final static Logger LOGGER = Logger.getLogger(DataContentViewerOtherCases.class.getName());
```

Comment:
Please revert back to lower-case "logger", even if codacy/netbeans complains about it.

Reply:
Reverted.